src/components/Header/NavBar.js

```diff
@@ -0,0 +1,37 @@
+import Link from 'next/link';
```

Comment:
I am seeing two different import approaches:
- `next/<item>`
- `@/path/to/item`

Which are we going with? I would like consistency, regardless of which we choose.

@istoga @Sheyitofunmi @TaylorDJones11 @ibimina @Domicaren1 @Achele

Reply:
next/<item> is a component/element import from nextjs e.g Image, head, Link e.t.c
while @/path/to/item is a component import from a src directory